support-frontend/assets/pages/weekly-subscription-landing/components/weeklyProductPrices.tsx

```diff
@@ -90,6 +90,14 @@ const weeklyProductProps = (
 		componentType: 'ACQUISITIONS_BUTTON' as OphanComponentType,
 	};
 
+	// TODO: remove/review this when the 12 for 12 offer is over
+	const isSpecialOffer = promotion?.promoCode === '12for12';
```

Comment:
please can this be `promotion?.promoCode.startsWith('12for12')`?
There will be a separate promo for Aus

Reply:
applied, thanks.